test/test_argument_parser.py

```diff
@@ -92,11 +92,11 @@ def test_argument_parser_decorator():
 
     # __getattr__
     decorator = ArgumentParserDecorator(parser)
-    assert decorator.add_argument == parser.add_argument
+    assert decorator.format_help == parser.format_help
```

Comment:
Note that `format_help` was chosen arbitrarily, since `add_argument` is now a defined method.